hypothesis-python/tests/cover/test_composite.py

```diff
@@ -120,9 +128,8 @@ class MyList(list):
 @given(st.data(), st.lists(st.integers()).map(MyList))
 def test_does_not_change_arguments(data, ls):
     # regression test for issue #1017 or other argument mutation
-    @st.composite
-    def strat(draw, arg):
-        return arg
+    def strat(arg):
+        return st.just(arg)
```

Comment:
Let's keep the old test, and just insert a `draw(st.none())` to keep the check happy - I think this would break the regression test as-is.